lib/pages/new_group/new_group.dart

```diff
@@ -42,7 +42,7 @@ class NewGroupController extends State<NewGroup>
   final createNewGroupChatInteractor = getIt.get<CreateNewGroupChatInteractor>();
   final contactStreamController = StreamController<Either<Failure, GetContactsSuccess>>();
   final groupNameTextEditingController = TextEditingController();
-  final uploadAvatarNewGroupChatNotifier = ValueNotifier<Either<Failure, Success>?>(null);
+  final avatarNotifier = ValueNotifier<AssetEntity?>(null);
   final createRoomStateNotifier = ValueNotifier<Either<Failure, Success>?>(null);
```

Comment:
why we need nullable here? Do you think it is harder to handle the `eventState`? Why not set it with a `initialState`?

Reply:
@hoangdat null mean user haven't select avatar yet, so no need to display avatar and upload it

Reply:
yes, but it is also hard for us to handle the `either`. Why not to specific it a state mean `user not yet do anything`.

Reply:
done, I created new `CreateNewGroupInitial` class